docs/includes/teams-support-note.md

```diff
@@ -0,0 +1,2 @@
+> [!IMPORTANT]
+> When using Excel with Teams, Office Scripts are only supported in Teams for the web (not Teams for Desktop, Mac, iOS, or Android).
```

Comment:
I think Desktop can represent Windows and Mac. Also, changing "for" to "on".
Suggestion:
When using Excel with Teams, Office Scripts are only supported in Teams on the web (not Teams on Windows, Mac, iOS, or Android).

Reply:
Good feedback. I've made the changes in this PR: https://github.com/OfficeDev/office-scripts-docs/pull/174/files